docs/releases/v0.25.0.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+## [v0.25.0](https://github.com/aws-observability/aws-otel-collector/tree/v0.25.0) (2023-01-05)
+
+[Full Changelog](https://github.com/aws-observability/aws-otel-collector/compare/v0.23.1...v0.25.0)
+
+**Closed issues:**
+
+- ECS EC2 sidecar deployment Cloudformation template [\#1591](https://github.com/aws-observability/aws-otel-collector/issues/1591)
+
+**Merged pull requests:**
+
+- Fixed incorrect docker-compose command [\#1730](https://github.com/aws-observability/aws-otel-collector/pull/1730) ([jkclark](https://github.com/jkclark))
+- Update GitHub Workflows to use IAM Roles w/ GitHub OIDC [\#1714](https://github.com/aws-observability/aws-otel-collector/pull/1714) ([bryan-aguilar](https://github.com/bryan-aguilar))
+- Update dependencies [\#1710](https://github.com/aws-observability/aws-otel-collector/pull/1710) ([bryan-aguilar](https://github.com/bryan-aguilar))
+- Updates to ECS EC2 sidecar deployment CFN template [\#1592](https://github.com/aws-observability/aws-otel-collector/pull/1592) ([PaurushGarg](https://github.com/PaurushGarg))
+
```

Comment:
I believe this is missing an entry for the updating of dependencies to `v0.68.0`?

Reply:
Updated.